static/js/chart/draw.ts

```diff
@@ -938,11 +1021,192 @@ function drawGroupBarChart(
     dataGroups[0].value.map((dp) => ({
       label: dp.label,
       link: dp.link,
-    }))
+    })),
+    options?.apiRoot
   );
   svg.attr("class", ASYNC_ELEMENT_CLASS);
 }
 
+/**
+ * Draw horizontally stacked bar chart.
+ * @param containerElement
+ * @param chartWidth
+ * @param dataGroups
+ * @param formatNumberFn
+ * @param options
+ */
+function drawHorizontalBarChart(
+  containerElement: HTMLDivElement,
+  chartWidth: number,
+  dataGroups: DataGroup[],
+  formatNumberFn: (value: number, unit?: string) => string,
+  options?: {
+    apiRoot?: string;
+    stacked?: boolean;
+    style?: {
+      barHeight?: number;
+      yAxisMargin?: number;
+    };
+    unit?: string;
+  }
```

Comment:
define the option type in static/js/chart/types.ts ?

Reply:
done